app/controllers/graders_controller.rb

```diff
@@ -286,4 +304,15 @@ def unassign_graders(grader_membership_ids)
     end
     Grouping.unassign_tas(grader_membership_ids, grouping_ids, @assignment)
   end
+
+  def find_empty_submissions(grouping_ids)
+     grouping_ids.each do |grouping_id|
+      submission = Submission.find_by_grouping_id(grouping_id)
+      if !submission || !SubmissionFile.where(submission_id: submission.id).exists?
+        grouping_ids.delete(grouping_id)
+        @found_empty_submission = true;
+      end
+    end
```

Comment:
`end` at 315, 4 is not aligned with `grouping_ids.each do |grouping_id|` at 309, 5.